omegaconf/omegaconf.py

```diff
@@ -679,6 +679,10 @@ def select(
     ) -> Any:
         try:
             try:
+                if cfg._parent is not None:
```

Comment:
I think this will make `select` behave differently for root vs non-root nodes.

Reply:
That's exactly the point.